app/controllers/pupils/digital_signage_controller.rb

```diff
@@ -24,7 +24,7 @@ class PublicDisplaysController < ApplicationController
     end
 
     def index
```

Comment:
I think this method can be removed, as I don't think there is a route for it anymore?